src/main/java/com/esri/samples/na/offline_routing/OfflineRoutingSample.java

```diff
@@ -97,6 +99,15 @@ public void start(Stage stage) throws Exception {
       routeTask = new RouteTask("./samples-data/san_diego/sandiego.geodatabase", "Streets_ND");
       routeTask.loadAsync();
 
+      // add a graphics overlay to show the boundary
+      Envelope envelope = new Envelope(new Point(-13045352.223196, 3864910.900750, 0, SpatialReferences
```

Comment:
@john0005 Yep, I found the same thing. There's no API to get the extent of a local geodatabase/routetask (only ones that are sync enabled it looks like). That's why I'm getting the points manually for now.